src/OnboardingSPA/components/Drawer/DrawerPanel/index.js

```diff
@@ -64,6 +66,7 @@ const DrawerPanel = () => {
 	}, [ drawerView ] );
 
 	return (
+		// eslint-disable-next-line jsx-a11y/no-static-element-interactions
```

Comment:
Let us do this like we have done [here](https://github.com/newfold-labs/wp-module-onboarding/blob/4406cb9de1ef268935d340ed60217c3e8350acd4/src/OnboardingSPA/components/Drawer/DrawerPanel/DesignColors.js#L238C6-L241C51).